ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/HeartbeatAgent.java

```diff
@@ -494,4 +515,16 @@ public String toString() {
       return String.valueOf(timestamp);
     }
   }
+
+  /**
+   * A listener for heartbeat related events.
+   */
+  public interface HeartbeatListener {
```

Comment:
HostStatusListener?  

nit:  we use Node in the KsLocator code.. and use Host here? might be good to use one?

Reply:
I guess I was thinking there might be other events, but maybe that was overthinking it.  Changed to HostStatusListener.

That's not a bad idea to factor out a single host entity, though I think Vicky is adding status to Node, and I'm planning on adding Lag, so Node will represent the joining of all of this data.  Maybe it will make sense to revisit that idea after the routing changes are made.